### PR TITLE
Fill in TODOs in documentation, fix alphabetical order in examples

### PR DESCRIPTION
**Pull request by erich666**

### CODE REVIEW

File: docs/api/lights/DirectionalLightShadow.html [hunk 1]
```diff
@@ -12,7 +12,7 @@
 
 		<h1>[name]</h1>
 
-		<div class="desc">TODO</div>
+		<div class="desc">Directional light with a shadow map, for casting shadows.</div>
```
Comment (mrdoob) [inline docs/api/lights/DirectionalLightShadow.html]: That's not correct. `DirectionalLightShadow` is a type of shadow (basically a camera a few more parameters).

Comment (erich666) [inline docs/api/lights/DirectionalLightShadow.html]: ... and this is why one should never accept pull requests that have "TODO" in the documentation. I'll go fix it and add to the PR.

**Comment by erich666** [comment]
@mrdoob Please do note new wording for the shadow map descriptions. Is it reasonable now? I'm here for another week and a few days before a 3 week-ish vacation, so wanted to ask now.

**Comment by mrdoob** [comment]
Sorry I didn't get to merge this at the time. Do you want to pull from `dev` and resolve conflicts or create a new PR?

**Comment by Mugen87** [comment]
In the meanwhile, docs were update by @looeee. The mentioned TODOs are resolved.

**Pull request closed without merging**